README.md

```diff
@@ -51,9 +51,11 @@ In order to use Astronomer, you'll need a GitHub token with `repo` read rights.
 
 ### Docker image
 
-Run `docker pull ullaakut/astronomer`.
+Run the astronomer docker image like such:
 
-Then, use the astronomer docker image like such: `docker run -t -e GITHUB_TOKEN=$GITHUB_TOKEN -v "/path/to/your/cache/folder:/data/" ullaakut/astronomer repositoryOwner/repositoryName`
+~~~bash
```

Comment:
I would suggest using backquotes for the code block instead of `~` if that's okay with you, for consistency with the rest of the markdown :)

Reply:
Done! :)